image/Dockerfile

```diff
@@ -12,23 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:stretch
+# --------------------------------------------------------------------------------------------------
+# The building stage.
+FROM golang:1.14.2-stretch AS builder
 
-RUN apt-get update
-RUN apt-get install -y curl ca-certificates
+WORKDIR /build/minibroker
+COPY ./go.mod ./go.mod
+COPY ./go.sum ./go.sum
+RUN go mod download
+
+COPY ./ ./
+RUN make build-linux
+
+# --------------------------------------------------------------------------------------------------
+# The downloading stage.
+FROM alpine:latest AS downloader
 
 ENV HELM_VERSION="v2.13.1"
 ENV FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
```

Comment:
Should these turn into build args instead?

Reply:
I'll have a follow-up PR with configurable args, they will include the image bases and this one.